zanata-war/src/test/resources/arquillian.xml

```diff
@@ -12,7 +12,7 @@
   <container qualifier="jbossas-managed">
     <configuration>
       <!-- We can use the JBOSS_HOME env variable instead of this -->
-      <property name="jbossHome">${arquillian.jboss.home}</property>
+      <property name="jbossHome">${jboss.home}</property>
```

Comment:
I honestly can't remember that discussion. In any case, I'm good with it now :smile: